### DISCUSSION TITLE
2013-6 - 泛函分析基础：学习指导和习题详解

### DISCUSSION THREAD

**Discussion by giscus** [duskmoon314/THU_BFA#5] [Announcements]
# 2013-6 - 泛函分析基础：学习指导和习题详解

https://duskmoon314.github.io/THU_BFA/ch6/2013-6.html

**Comment by Timothy-Liuxf** [comment]
$\lVert Tx_n\rVert \leq C$ 似乎不能推出 $\lVert T \rVert \leq C$？
这道题不需要用到典范映射和一致有界性原理，直接用闭图像定理就可以做出来了。

**Reply by Timothy-Liuxf** [reply]
$\lVert Tx_n\rVert \leq C$ 这个公式怎么渲染不出来了（

**Reply by duskmoon314** [reply]
可以直接 `\|` 的（

**Reply by thulanxc** [reply]
这个应该是需要验证所有的收敛列吧，不只是验证收敛到0的收敛列。不过由于T是线性的，似乎是等价的。

**Reply by duskmoon314** [reply]
应该可以构造任意收敛列为 $x + 0$ 的样子？
